pages/getting-started/understand-odata-in-6-steps.html

```diff
@@ -1351,4 +1351,26 @@ <h4>Contribute to "Understanding OData in 6 steps"</h4>
 </div>
 </p></div>
 </p></div>
-</p></div>
\ No newline at end of file
+</p></div>
+<script>
+    $(document).ready(function() {
+       $('#step-collection-modal').on('shown.bs.modal', function () {
```

Comment:
Since you are binding on the same event use the simplified jquery syntax for binding events to multiple elements 
```javascript
$('#element1, #element2, #element3').on('event', function (){ 
//
})
```